packages/story-editor/src/components/library/panes/text/textSets/textSetsPane.js

```diff
@@ -208,7 +208,7 @@ function TextSetsPane({ paneRef }) {
     <SectionContainer id={sectionId}>
       <TitleBar>
         <Headline
-          as="h3"
+          as="h2"
```

Comment:
there's no `h2` between the `h1` and this as an `h3` which is skipping a header level